mlir/test/Conversion/TosaToLinalg/tosa-to-linalg.mlir

```diff
@@ -100,6 +100,26 @@ func.func @test_add_0d(%arg0: tensor<f32>, %arg1: tensor<f32>) -> tensor<f32> {
 
 // -----
```

Comment:
Should you add e.g. a partner test to `divsi` for the unsigned type?

Reply:
done